### PR TITLE
81 # Wire in document title input

### PR DESCRIPTION
**Pull request by fructusmortus**

### CODE REVIEW

File: module-3/web-app/src/components/EditorContainer/EditorTitle/EditorTitle.jsx [hunk 1]
```diff
@@ -2,10 +2,17 @@ import React from "react"
 import style from "./EditorTitle.module.scss"
 
 
-const EditorTitle = () => {
+const EditorTitle = ({onChange, documentTitle}) => {
+
   return (
-        <div><input type="text" name="title" placeholder="Enter your title here..."/></div>
+        <div><input
```
Comment (shelby-carter) [inline module-3/web-app/src/components/EditorContainer/EditorTitle/EditorTitle.jsx:8]: Is there a real benefit of wrapping input into a div?

File: module-3/web-app/src/components/EditorContainer/EditorContainer.jsx [hunk 2]
```diff
@@ -1,13 +1,55 @@
...
+            dispatch(readDocument(documentId))
+        }
+    }, [])
+
     return (
         <div>
-            <EditorTitle/>
-            <ProseMirror documentId={documentId}/>
+            <EditorTitle onChange={onChange} documentTitle={documentTitle}/>
+            <ProseMirror onEditorContentUpdate={onEditorContentUpdate} documentTitle={documentTitle.title}/>
```
Comment (shelby-carter) [inline module-3/web-app/src/components/EditorContainer/EditorContainer.jsx:52]: Pass down document content if document is in edit mode

File: module-3/web-app/src/components/EditorContainer/EditorTitle/EditorTitle.jsx (hunk 1 above)
Comment (fructusmortus) [reply module-3/web-app/src/components/EditorContainer/EditorTitle/EditorTitle.jsx]: Fixed

**Comment by shelby-carter** [comment]
LGTM

**Pull request merged**